exchanges/okex/okex_wrapper.go

```diff
@@ -197,10 +197,11 @@ func (o *OKEX) SetDefaults() {
 		// TODO: Specify each individual endpoint rate limits as per docs
```

Comment:
Review: `[ERROR]|TICKER| 29/12/2020 16:22:24 |Failed to get REST ticker. Error: ticker fetching not supported for index`
If there is no index support for ticker just resolve.